appstream.patch

```diff
@@ -0,0 +1,28 @@
+Date: Fri, 10 May 2024 16:40:35 +0000
+Subject: [PATCH] Add developer name and launchable.
+
+---
+ org.kde.sweeper.appdata.xml | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/org.kde.sweeper.appdata.xml b/org.kde.sweeper.appdata.xml
+index d2dcd0a..f75b3fc 100644
+--- a/org.kde.sweeper.appdata.xml
++++ b/org.kde.sweeper.appdata.xml
+@@ -3,6 +3,7 @@
+   <id>org.kde.sweeper.desktop</id>
```

Comment:
@bbhtt does this `id` need to be changed?

Reply:
Yes https://invent.kde.org/teams/flathub/issues/-/issues/28#error-appstream-id-mismatch-flatpak-id